### ISSUE TITLE
(URGENT) All tests failing due to lutaml-model 0.3.23

### ISSUE THREAD

**Issue by ronaldtse** [lutaml/ogc-gml#12]
All tests are failing now. All tests used to pass with 0.3.10.

e.g.

```
  1) Ogc::Gml::CompositeCurve round-trips CompositeCurve.xml
     Failure/Error:
       output = Ogc::Gml::CompositeCurve.from_xml(input).to_xml(
         pretty: true,
         declaration: true,
         encoding: "utf-8"
       )
     
     NoMethodError:
       undefined method `pos=' for an instance of Ogc::Gml::AbstractCurveSegment
     # ./spec/ogc/gml/composite_curve_spec.rb:12:in `block (2 levels) in <top (required)>'
```

XML:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<gml:CompositeCurve gml:id="CompositeCurve-1"
	xmlns:gml="http://www.opengis.net/gml/3.2">
	<gml:curveMember>
		<gml:Curve gml:id="Curve-1" srsName="urn:ogc:def:crs:EPSG::4326">
			<gml:segments>
				<gml:Arc interpolation="circularArc3Points">
					<gml:pos>49.194442 -122.929795</gml:pos>
					<gml:pos>49.040829 -123.086952</gml:pos>
					<gml:pos>49.361246 -123.183889</gml:pos>
				</gml:Arc>
			</gml:segments>
		</gml:Curve>
	</gml:curveMember>
<!-- ... -->
```

```ruby
module Ogc
  module Gml
    class AbstractCurveSegment < Lutaml::Model::Serializable
      attribute :num_derivatives_at_start, :integer
      attribute :num_derivatives_at_end, :integer
      attribute :num_derivative_interior, :integer

      xml do
        root "AbstractCurveSegment"
        namespace "http://www.opengis.net/gml/3.2", "gml"

        map_attribute "numDerivativesAtStart", to: :num_derivatives_at_start
        map_attribute "numDerivativesAtEnd", to: :num_derivatives_at_end
        map_attribute "numDerivativeInterior", to: :num_derivative_interior
      end
    end
  end
end
```

Notice that `pos=` is an attribute inside the `Arc` object, which does have `pos` as an attribute:

```ruby
module Ogc
  module Gml
    class Arc < ArcString
      xml do
        root "Arc"
        namespace "http://www.opengis.net/gml/3.2", "gml"

        map_attribute "numDerivativesAtStart", to: :num_derivatives_at_start
        map_attribute "numDerivativesAtEnd", to: :num_derivatives_at_end
        map_attribute "numDerivativeInterior", to: :num_derivative_interior
        map_attribute "interpolation", to: :interpolation
        map_attribute "numArc", to: :num_arc
        map_element "pos", to: :pos
        map_element "pointProperty", to: :point_property
        map_element "pointRep", to: :point_rep
        map_element "posList", to: :pos_list
        map_element "coordinates", to: :coordinates
      end
    end
  end
end
```

**Comment by HassanAkbar** [comment]
@ronaldtse Initially it looks like this is happening because of [this issue](https://github.com/lutaml/lutaml-model/issues/153) in `lutaml-model`. I will try to fix it as soon as possible.

**Comment by ronaldtse** [comment]
Confirm fixed in 0.3.24. Thanks @HassanAkbar !
